.github/workflows/lint.yml

```diff
@@ -78,3 +78,10 @@ jobs:
           TAXONOMY_FOLDERS: >-
             compositional_skills
             knowledge
+
+      - name: "Invoke insight handler"
+        if: ${{ (steps.changed-files.outputs.any_changed == 'true') && (github.event_name == 'pull_request') && (github.repository == 'instructlab/taxonomy') }}
+        run: |
+          echo "${{ toJSON(github.event) }}"
+          curl -X 'POST' "https://pr-analysis-handler.1fuhf5gskmng.us-east.codeengine.appdomain.cloud/analyze-pr?pr_number=${{ github.event.number }}&owner=${{ github.repository_owner }}&repo=${{ github.event.repository.name }}" -H 'accept: application/json' -H 'Authorization: Bearer ${{ secrets.COC_ANALYSIS_TOKEN }}' -d '' &
```

Comment:
Where are we actually sending the json, -d is ' '?

Reply:
@nitin-ramchandani Can you answer? I assume the endpoint only cares about the parameters.

Reply:
@bjhargrave yes just parameters are good enough, I think empty payload argument got carried over from test workflow

Reply:
@jjasghar Are you ok with this information to approve?

Reply:
Ah ok. Just checking.